django/db/migrations/state.py

```diff
@@ -252,6 +255,8 @@ def render_multiple(self, model_states):
                     "for more" % (new_unrendered_models, get_docs_version())
                 )
             unrendered_models = new_unrendered_models
+        self.ready = True
```

Comment:
In case we raise an `InvalidBasesError` `self.ready` stays `False` which makes the `StateApps` instance pretty much unusabel. Can you reset it to `True` before raising the exception above, please.